snap/snapcraft.yaml

```diff
@@ -96,6 +96,8 @@ apps:
     command: microk8s-cilium.wrapper
   juju:
     command: microk8s-juju.wrapper
+  dbctl:
+    command: microk8s-dbctl.wrapper
```

Comment:
If you have time, can you please consider which plugs would be needed for strict?

Reply:
I think we would need this https://snapcraft.io/docs/personal-files-interface to be able to write the backup file.